scripts/run_tests.py

```diff
@@ -712,17 +719,28 @@ def compile_tests_to_project(trg_proj,
                              applet_folder,
                              compiler_flags,
                              version_id,
-                             lazy_flag):
+                             lazy_flag,
```

Comment:
Please update the docstring (comment) on this function

Reply:
done